lint-tasks.yml

```diff
@@ -97,6 +139,38 @@ tasks:
           black --color --line-length 100 {{.BLACK_FLAGS}} .
           ruff check {{.RUFF_FLAGS}} .
 
+  linter-nodejs:
+    internal: true
+    vars:
+      CHECKSUM_FILE: "{{.BUILD_DIR}}/{{.TASK}}.md5"
+      OUTPUT_DIR: "{{.LINTER_NODEJS_BUILD_DIR}}"
+    cmds:
+      - task: ":nodejs"
+        vars:
+          NODEJS_VERSION: "latest"
+          OUTPUT_DIR: "{{.OUTPUT_DIR}}"
+      # Checksum the generated files (this command must be last)
+      - >-
+        tar cf - --directory "{{.OUTPUT_DIR}}" {{.CHECKSUM_TAR_BASE_ARGS}} .
+        | md5sum > "{{.CHECKSUM_FILE}}"
+    sources:
+      - "{{.TASKFILE_DIR}}/lint-tasks.yml"
+      - "{{.TASKFILE_DIR}}/Taskfile.yml"
+    status:
+      - "test -f '{{.CHECKSUM_FILE}}'"
+      - >-
+        diff
+        <(tar cf - --directory "{{.OUTPUT_DIR}}" {{.CHECKSUM_TAR_BASE_ARGS}} . | md5sum)
+        "{{.CHECKSUM_FILE}}"
+
+  node-modules:
```

Comment:
* This task should depend on the linter-nodejs checksum, right?
* If we checksum this task, we should add package.json to the `sources`